README.adoc

```diff
@@ -86,83 +86,45 @@ include::{common-includes}/gitclone.adoc[]
 ----
 
-By default, the Open Liberty runtime is downloaded to the `/opt/ol/wlp` directory.
+The `FROM` instruction initializes a new build stage and indicates the parent image from which your
+image is built. If you don't need a parent image, then use `FROM scratch`. This makes your image
+a base image. In this case, you're using the `openliberty/open-liberty:javaee7` image are your parent
+image, which comes with the latest Open Liberty runtime.
 
-Next, create a symlink between the `/opt/ol/wlp/usr/servers` directory and the `/servers` directory.
-This symlink makes mounting your servers easier because you won't have to type long path names:
+The `RUN` instruction executes various shell commands in a new layer on top of the current image.
+In this case, you're creating a symlink between the `/opt/ol/wlp/usr/servers` directory and the `/servers`
+directory. This makes mounting your servers easier because you won't have to use long path names.
 
-[source, indent=0]
-----
-include::finish/Dockerfile[tags=link]
-----
+The `CMD` and `ENTRYPOINT` instructions define a default command that executes when the image runs as
+a container. The only difference between the two is that `CMD` will be overridden with any arguments
+passed at the end of the `docker run` command while `ENTRYPOINT` requires the `--entrypoint` flag to
+be overridden. In this case, you're using the `ENTRYPOINT` instruction to start a Open Liberty server
+and the `CMD` instruction to indicate which server to start. The `CMD` instruction makes it convenient
+to start any server you want since its easily overridable.
 
-Although you have only one `RUN` instruction in your Dockerfile, you can tie multiple
-`RUN` instructions together to minimize the number of layers in your image and reduce build time. While
-maintaining readability, try to minimize the number of layers in your image, as shown in the following example:
+For a complete list of available instructions, see the official https://docs.docker.com/engine/reference/builder[Docker documentation].
 
-[role="no_copy"]
-----
-RUN mkdir dir1 \
-    && mv dir1 dir2 \
-    && cp dir2 dir3
-----
-
-Finally, use the `ENTRYPOINT` instruction to run the `server` script and use the `CMD` instruction to
-specify which server to start. Because a user can override `CMD` calls, you can override the
-`defaultServer` name with another server name when running the container:
-
-[source, indent=0]
-----
-include::finish/Dockerfile[tags=default-start]
-----
+=== Optional: Writing a .dockerignore file
 
-Your Dockerfile is complete. Use it to build your image.
+When Docker runs a build, it sends all of the files and directories located in the same directory as the
+Dockerfile to its build context, making them available for use in instructions like `ADD` and `COPY`.
+To speed up the build, exclude all of the files and directories that aren't necessary for building
+your image by adding them to a `.dockerignore` file.
```

Comment:
"To make the build faster, exclude all of the files and directories that aren't necessary for building your image. Add them to a `.dockerignore` file."

Reply:
Maybe "To make image building faster, add all files and directories that aren't necessary for building your image to a ``.dockerignore` file. This excludes them from the build context."?